### PR TITLE
chore: update doc

### PR DESCRIPTION
**Pull request by yuqizhou77**
- Update "switch M365 account" to "switch M365 tenant"
- Also add workarounds for projects launched in Visual Studio

### CODE REVIEW

**Review by MuyangAmigo — DISMISSED** [review]
LGTM

File: docs/fx-core/switch-tenant-or-subscription-help.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This doc is to help you understand what will happen when provisioning in an alre
 
 ## Switch Microsoft 365 Account
 ### Local Debug
-You could run local debugging for a Teams project with one M365 account and then easily switch to another account for further local debugging. To do this, you only need to:
+You could run local debugging for a Teams project with one M365 tenant and then easily switch to another tenant for further local debugging. To do this, you only need to:
```
Comment (zyxiaoyuer) [inline docs/fx-core/switch-tenant-or-subscription-help.md:7]: Do not use words M365 but use Microsoft 365, I need to create a bug for this. recently got feedback from M365 developer program PM that we should never use M365 anywhere.

File: docs/fx-core/switch-tenant-or-subscription-help.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ After that, we will
 3. `state.local.json` file in .fx/states folder will be overwritten with the information of new resources in the new M365 tenant. If the project requires AAD, `local.userdata` will be overwritten with the new client secret.
 
 ### Provision in a Remote Environment
-You could provision resources in a remote environment with one M365 account and then re-provision in the same environment but with another M365 account. To do this, you only need to:
+You could provision resources in a remote environment with one M365 tenant and then re-provision in the same environment but with another M365 tenant. To do this, you only need to:
```
Comment (zyxiaoyuer) [inline docs/fx-core/switch-tenant-or-subscription-help.md:18]: Microsoft 365 again.

File: docs/fx-core/switch-tenant-or-subscription-help.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ After that, we will
...
 1. Sign out of the current M365 account.
```
Comment (zyxiaoyuer) [inline docs/fx-core/switch-tenant-or-subscription-help.md:19]: same here.

Comment (yuqizhou77) [reply docs/fx-core/switch-tenant-or-subscription-help.md]: updated.

File: docs/fx-core/switch-tenant-or-subscription-help.md (hunk 2 above)
Comment (yuqizhou77) [reply docs/fx-core/switch-tenant-or-subscription-help.md]: updated.

File: docs/fx-core/switch-tenant-or-subscription-help.md (hunk 1 above)
Comment (yuqizhou77) [reply docs/fx-core/switch-tenant-or-subscription-help.md]: Fixed.

**Review by zyxiaoyuer — APPROVED** [review]

File: docs/fx-core/switch-tenant-or-subscription-help.md [hunk 4]
```diff
@@ -1,28 +1,28 @@
-This doc is to help you understand what will happen when provisioning in an already-provisioned environment but with different account or Azure subscription or local debugging again with another Microsoft 365(M365) account. We will also explain how to recover from the backups in this doc.
+This doc is to help you understand what will happen when provisioning in an already-provisioned environment but with different account or Azure subscription or local debugging again with another Microsoft 365(Microsoft 365) account. We will also explain how to recover from the backups in this doc.
```
Comment (MuyangAmigo) [inline docs/fx-core/switch-tenant-or-subscription-help.md:2]: `Microsoft 365(Microsoft 365)` has duplicates.

Comment (yuqizhou77) [reply docs/fx-core/switch-tenant-or-subscription-help.md:2]: Thanks for catching this! Fixed.

**Review by MuyangAmigo — APPROVED** [review]

**Pull request merged**